src/libcore/array/mod.rs

```diff
@@ -1,5 +1,5 @@
 //! Implementations of things like `Eq` for fixed-length arrays
-//! up to a certain length. Eventually we should able to generalize
+//! up to a certain length. Eventually we should be able to generalize
```

Comment:
```suggestion
//! up to a certain length. Eventually we should generalize
```
Could also maybe be this but this "feels" different 🤷‍♂